spec/rspec/mocks/exclude_stubbed_classes_from_subclasses_spec.rb

```diff
@@ -0,0 +1,68 @@
+if RUBY_VERSION >= '3.1'
+  class TestClass
+  end
+
+  module RSpec
+    module Mocks
+      RSpec.describe ExcludeStubbedClassesFromSubclasses do
+        after do
+          described_class.disable!
+        end
+
+        describe '.enable!' do
+          it 'does not extends Class when it has been enabled' do
+            described_class.enable!
+
+            expect {
+              described_class.enable!
+            }.not_to(change { Class.respond_to?(:subclasses_with_rspec_mocks) })
```

Comment:
I don’t quite understand this. We enable the e feature, the Class is altered, and we expect the second call to enable not to … remove the method added by previous enable?

Reply:
I agree this wasn't making sense. I changed.